src/components/information-box.ts

```diff
@@ -71,4 +71,15 @@ export default class InformationBoxControl extends Control {
     super({ element: infoBox});
     this.div = infoBox;
   }
+
+  public show() {
+    this.div.classList.remove('fade-out');
+    this.div.classList.remove('disabled');
```

Comment:
I would remove this line from that one since disabled seems only used in notification.ts

Reply:
Right, I remove the method but not the CSS, my bad